### PR TITLE
add env for esrf

### PR DESCRIPTION
**Pull request by michael-boese**
Environment for esrf.

### CODE REVIEW

File: interfaces/tango/configs.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+name: tango
+version: "0.1"
+dependencies:
+  - badger-opt
+  - numpy
```
Comment (wenatuhs) [inline interfaces/tango/configs.yaml:6]: Might want to add `tango` python package in the dependencies list

File: interfaces/tango/README.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# TANGO_MOCK Interface for Badger
```
Comment (wenatuhs) [inline interfaces/tango/README.md:1]: I know this is a placeholder but you might want to change the title to like "TANGO Interface for Badger" lol

Actually just ignore this comment lol

**Review by wenatuhs — COMMENTED** [review]
Looks good! And you are already a Badger master since you are using the AOE version of `set_var` :)

File: interfaces/tango/README.md (hunk 2 above)
Comment (michael-boese) [reply interfaces/tango/README.md:1]: > I know this is a placeholder but you might want to change the title to like "TANGO Interface for Badger" lol
> 
> Actually just ignore this comment lol

**Review by wenatuhs — APPROVED** [review]

**Pull request merged**